docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,529 @@
+# Policy On Namespace
+
+- Status: accepted
+
+## Context and Problem Statement
+
+Kuma 2.6.0 introduced a new feature allowing users to apply policies on Zone CP.
+This change unblocks us to introduce a namespace support for policies.
+The CRDs for new policies are already namespace-scoped, but they can be applied only on the `kuma-system` namespace.
+
+## Considered Options
+
+- allow applying new policies to the custom namespaces
+- do nothing
+
+## Decision Outcome
+
+- allow applying new policies to the custom namespaces
+
+## Pros and Cons of the applying policies on custom namespaces
+
+* Good, because allows using Kubernetes RBAC
+* Good, because provides more predictable behaviour as namespace-scoped policy affects only workloads in the same
+  namespace
+* Good, because provides more Kubernetes-native UX
+* Bad, because adds complexity for users
+    * what's the right namespace for the policy?
+    * how cross-policy refs work?
+    * how policy order works?
+
+## Pros and Cons of doing nothing
+
+* Good, because we have time to work on other features
+* Bad, because cluster operator has to grant write permissions to `kuma-system` namespace to everyone who works with
+  policies
+* Bad, because low isolation between teams (team-a can unintentionally break polices of the team-b)
+
+## Implementation
+
+Applying a policy on the custom namespace **affects requests to the pods in the custom namespace**.
+Since some policies are applied on the client-side, the scope of policy's effect is not limited to the namespace
+where it was applied. 
```

Comment:
Why? The first sentence says policy in the custom namespaces affects **requests** to the custom namespace (not pods in the custom namespace). And the second sentence just elaborates – since some policies are applied on the client-side the policy scope **is no longer limited** to policy's namespace.

Reply:
I think I didn't see requests. I had to reread this 3 times, I think it could be made more clear